Lecture-4/Task4/l4task4/TimeRunnersComparator.java

```diff
@@ -0,0 +1,12 @@
+package l4task4;
+
+import java.util.Comparator;
+
+public class TimeRunnersComparator implements Comparator<Runner> {
```

Comment:
Is it ascending or descending time order? This should be specified in a class name

Reply:
done